### PR TITLE
Update all references to the console on port 8444

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@smarterclayton @bparees

File: README.md [hunk 1]
```diff
@@ -161,6 +161,14 @@ OpenShift and Kubernetes integrate with the [Swagger 2.0 API framework](http://s
 
 and then browse to http://openshift3swagger-claytondev.rhcloud.com (which runs a copy of the Swagger UI that points to localhost:8080 by default).  Expand the operations available on v1beta1 to see the schemas (and to try the API directly).
 
+Management Console
+------------------
+
+The OpenShift API server also hosts the web-based management console. You can try out the management console at [http://localhost:4443/console](http://localhost:4443/console).
```
Comment (bparees) [inline README.md]: 4443?  8443?

**Comment by jwforres** [comment]
@bparees fixed

**Comment by bparees** [comment]
lgtm

**Comment by jwforres** [comment]
[merge]

**Pull request merged**